src/content/docs/apm/agents/c-sdk/install-configure/install-c-sdk-compile-link-your-code.mdx

```diff
@@ -26,7 +26,7 @@ Our C SDK auto-instruments your code so you can start monitoring applications. Y
 If you don't have one already, [create a New Relic account](https://newrelic.com/signup). It's free, forever.
 
 <ButtonLink
-  data-tessen="stitchedPathLinkClick"
+  data-nr-browser-agent="stitchedPathLinkClick"
```

Comment:
What if we just got rid of all of these?

Reply:
these buttons become `MDXButton` which calls `Button` in the theme and this is not one of the listed prop types. Also, I can't find `data-tessen` in any files that aren't mdx so i really don't think we need this

Reply:
yeah i didn't find any events associated with them either, i don't think they're hooked up to anything anymore. but.. do we want them to be? maybe they should be getting tracked?

Reply:
`stitchedLinkPath` is still actively being logged. Is there coming from somewhere else?

Reply:
I guess it's coming from the theme and the SignupLink. I don't am not familiar with what the point of this value is. if it's not hooked up, it does't seem like it's being used anymore.